lib/config/presets/internal/group.ts

```diff
@@ -678,7 +679,7 @@ for (const monorepo of Object.keys(monorepos.presets)) {
   };
 }
 config.monorepos = {
-  description: 'Group known monorepo packages together',
+  description: 'Group known monorepo packages together.', // Check me on on adding a full stop here
```

Comment:
Check me on this one as well. Remember to remove the comment!

Reply:
It's ok